automated/linux/ltp/skipfile-lkft.yaml

```diff
@@ -18,13 +18,31 @@
 # tests:
 #   List of tests to skip
 
+globals:
+  - boards: &all_boards
```

Comment:
This list is a list from the LAB at lkft.validation.linaro.org

Reply:
Added a comment to explain the devices are the ones available in the lab and tuxsuite.